packages/api-sdk/src/resources/base.ts

```diff
@@ -12,6 +12,14 @@ class BaseResource<Client extends Record<string, any>> extends Fetcher<Client> {
   protected _getFieldsQuery(fields?: Fields): string {
     return fields ? `?fields=${fields.join(',')}` : '';
   }
+
+  protected _getIDsQuery(ids: string[]): string {
```

Comment:
Can we use `URL` + `URLSearchParams` to build this instead? See: https://github.com/voiceflow/libs/pull/263#discussion_r871828767

Reply:
using the URL class doesn't really work that well since it requires the entire href. however, axios is already preconfigured to use the API baseurl, so all urls passed into axios are relative, which the URL class can't construct